protocol/src/main/java/org/opensearch/sql/protocol/response/format/ResponseFormatter.java

```diff
@@ -27,4 +27,6 @@ public interface ResponseFormatter<R> {
    */
   String format(Throwable t);
 
+  String getFormat();
```

Comment:
Add a javadoc comment for what it's used for

Reply:
added java doc in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/0df2a1b34b182cea23bfb490d7143bd726aa00d4